server/views/admin/placementRequests/index.njk

```diff
@@ -53,7 +53,7 @@
                 },
                 id: "requestType",
                 name: "requestType",
-                items: convertObjectsToSelectOptions(PlacementRequestUtils.requestTypes, 'All request types', 'name', 'value', 'requestType', context)
+                items: convertObjectsToSelectOptions(PlacementRequestUtils.requestTypes, 'All request types', 'name', 'value', 'requestType', '', context)
```

Comment:
It's a shame we can't have a default value here, but I guess it's tricky as we have to pass the context in. Not the end of the world though